shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -60,7 +60,10 @@ def clean_folder_name(folder_name: str) -> str:
     Returns:
     str: The cleaned folder name.
     """
-    folder_name = os.path.normpath(folder_name)
+    folder_name = folder_name.strip("/")
+    if folder_name != "":
```

Comment:
In my opinion it would be better to fail loudly in a scenario where a  None is explicitly set or any non string value. Do you feel strongly about handling `None` [](edited-with-Axolo)

Reply:
Give me your rationale for why it is better to fail in the instance of `None`? I’m not opposed, just want to hear your thought behind it [](edited-with-Axolo)

Reply:
the function Clean_folder_name is very specific and used in different scenarios. if you're explicitly passing in None it is most likely unintentionally from a variable that was expecting a return.
otherwise we're making an assumption it was intentional which is not a safe assumption in my opinion. [](edited-with-Axolo)